nncf/quantization/algorithms/bias_correction/onnx_backend.py

```diff
@@ -114,12 +114,13 @@ def get_output_name(model: onnx.ModelProto, node_name: str) -> List[str]:
 
     @staticmethod
     def is_quantized_weights(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        is_quantized_weights = False
         input_nodes = [edge.from_node for edge in nncf_graph.get_input_edges(node)]
-        if not hasattr(node.metatype, "weight_definitions"):
-            return False
-        weight_port_id = node.metatype.weight_definitions.weight_port_id
-        weight_node = input_nodes[weight_port_id]
-        return weight_node.metatype == ONNXDequantizeLinearMetatype
+        weight_port_ids = node.metatype.weight_port_ids
+        for weight_port_id in weight_port_ids:
+            weight_node = input_nodes[weight_port_id]
+            is_quantized_weights = is_quantized_weights or weight_node.metatype == ONNXDequantizeLinearMetatype
+        return is_quantized_weights
```

Comment:
I think we should move implementation to `nncf/onnx/graph/node_utils.py` and reuse it in BC and FBC algorithms

Reply:
Done